### PR TITLE
Chemist and Psychiatrist locker fixes, for real this time.

### PR DESCRIPTION
**Pull request by mistyLuminescence**
Previously, the chem locker had psych stuff (??), and the psych locker didn't actually use the psych locker (???), it was just a regular locker locked to psych access with psych stuff in it (????).

Now, the chem lockers contain chem-related things, and the psych locker is the actual psych locker instead of a regular locker with a psych access lock. 

Additionally, vials (the 30u virology ones) are now tiny, not small, items - because beakers and bottles are small and they're 60u (or 120u... or 300u...), and a vial is half a beaker or bottle.

Edit: merge conflicts suck, also fight me travis

Old chem locker: psych clothes, straightjacket, soporific, syringe, citalopram, methylphenidate, clipboard, folder, tape recorder and tape, camera, therapy doll.
Old psych locker: pill bottles.

New chem locker: pill bottles, beakers, autoinjectors, syringes, inaprov, dylovene, droppers.
New psych locker: everything in the old 'chem' locker.

### CODE REVIEW

File: polaris.dme [hunk 1]
```diff
@@ -361,7 +361,6 @@
 #include "code\game\antagonist\station\renegade.dm"
 #include "code\game\antagonist\station\revolutionary.dm"
 #include "code\game\antagonist\station\rogue_ai.dm"
-#include "code\game\antagonist\station\thug.dm"
```
Comment (MisterLayne) [inline polaris.dme]: Not entirely sure this is supposed to be removed(?)

Comment (mistyLuminescence) [reply polaris.dme]: DAMMIT MERGE CONFLICT

**Comment by mistyLuminescence** [comment]
good spot @MisterLayne , fixed

**Comment by Anewbe** [comment]
You don't seem to have used the map merger. Please go back and do so.

**Comment by mistyLuminescence** [comment]
DISREGARD

**Comment by mistyLuminescence** [comment]
it's actually a functioning PR now don't worry guys

**Comment by Anewbe** [comment]
Why does the Chem locker have a straight jacket in it?

**Comment by mistyLuminescence** [comment]
The old one had a straightjacket, and the new one has a straightjacket because it's the only thing in there that's even remotely hard to replace. Could just add another to secondary storage and remove this one, but  touching the map files again will probably break something...

**Comment by Anewbe** [comment]
Yeah, let's scrap that jacket. Chemistry does not need it.

**Comment by mistyLuminescence** [comment]
Straightjacket removed, let's remove all the other straight things too pls
this is the northern star after all

**Pull request merged**